src/main/java/Main.java

```diff
@@ -8,6 +9,16 @@ public static void main(String[] args) {
     }
 
     public void run(Scanner scanner) {
-        // napisz swój program tutaj. Do wczytywania danych użyj przekazanego w parametrze scannera
+        List<Integer> numbers = ListUtils.getNumbers(scanner);
+        try {
+            ListUtils.printNumbersInReverse(numbers);
+            ListUtils.printSumOfNumbersCalculation(numbers);
+            int lowest = ListUtils.getLowestNumber(numbers);
+            int greatest = ListUtils.getGreatestNumber(numbers);
+            System.out.println("Najmniejsza liczba w liście to " + lowest);
+            System.out.println("Największa liczba w liście to " + greatest);
+        } catch (IllegalArgumentException e) {
+            System.err.println(e.getMessage());
+        }
     }
```

Comment:
Unikaj sterowania programem wyjątkami, jest to używane jako antywzorzec: https://medium.com/@kritika.singhal/coding-by-exception-682060d8f0f1

Reply:
W tym przypadku możnesz zamiast try dodać ifa sprawdzającego czy lista niej jest pusta.